### ISSUE TITLE
Implement CDML Writer

### ISSUE THREAD

**Issue by Longor1996** [Longor1996/TCGE#2]
Implement a writer for the `.cdml`-format, given the format and data-structures defined in [/docs/cdml.ebnf](https://raw.githubusercontent.com/Longor1996/TCGE/master/docs/cdml.ebnf), preferably in the same module as the parser.

**Comment by Longor1996** [comment]
Currently blocked by #1

**Comment by Longor1996** [comment]
New plan: CSBS is effectively a multiversal scripting language, so serialization doesn't make sense unless one is talking about the Abstract Syntax Tree or some other intermediate form/syntax.

As such, this issue will be closed.
